<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>systemPreferences</class>
 <widget class="QDialog" name="systemPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>System Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QGroupBox" name="analysisTabLabelsGroupBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Analysis Tab Labels</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>116</width>
            <height>96</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="analysisTabLabelsVerticalLayout">
           <item>
            <widget class="QRadioButton" name="labelTabsWithWorkItemID">
             <property name="text">
              <string>Work Item ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="labelTabsWithIndividualID">
             <property name="text">
              <string>Patient ID</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="labelTabsWithSubjectID">
             <property name="text">
              <string>Subject ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="labelTabsWithPatientName">
             <property name="text">
              <string>Patient Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="labelTabsWithAlsoKnownAs">
             <property name="text">
              <string>Also Known As</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="generalGroupBox">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="encounterWindowLabel">
            <property name="text">
             <string>Encounter Window (days):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="encounterWindow">
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="savePromptPreferenceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Prompt Before Saving Changes:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="promptBeforeSave">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>(check to prompt)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="saveConfirmWhenSetProcessingParametersPreferenceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Seek Confirmation When Changing Processing Parameters:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="confirmWhenSetProcessingParameters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>(check to seek confirmation)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="savepresentCompositionControlPreferenceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Present Limit Controls After Computing Composition:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="presentCompositionControl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>(check to display the control)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="backdoorMessagePreferenceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Message when Extract Scripts are Run:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="displayExtractMessages">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>(check if want messages)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="importTab">
      <attribute name="title">
       <string>Import</string>
      </attribute>
      <widget class="QGroupBox" name="importGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>771</width>
         <height>301</height>
        </rect>
       </property>
       <property name="title">
        <string>Import</string>
       </property>
       <widget class="QDoubleSpinBox" name="recentlyImportedImages">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>37</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="recentlyImportedImagesLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>200</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Recently Imported Images (days)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="requireOriginalCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>441</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Require Pixel Data ORIGINAL</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="excludeLocalizerCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>451</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Exclude Image Specialization LOCALIZER</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="excludePreContrastCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>451</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Exclude Image Specialization PRE_CONTRAST</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="importFoldersRecursivelyCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>441</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Import Folders Recursively</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="viewerTab">
      <attribute name="title">
       <string>Viewer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QGroupBox" name="keyAssignmentsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Key Assignments</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="interactionGroupBox">
         <property name="title">
          <string>Interaction</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="targetDefineIntensityWindowLabel">
            <property name="text">
             <string>Initializer Intensity Window:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="targetDefineIntensityWindow">
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>400.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>systemPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>systemPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
